nuc_morph_analysis/analyses/density/confirm_no_corr_btn_density_and _nuclear_area.py

```diff
@@ -21,52 +22,19 @@
 dfm = filter_data.all_timepoints_minimal_filtering(df)
 
 #%% # plot density over time for each colony  along colony time 
-x_col = "colony_time"
-column_val = 'label_img'
-feature_list = ['2d_area_nuc_cell_ratio','density','2d_area_nucleus','2d_area_pseudo_cell',
-                '2d_area_cyto','inv_cyto_density','2d_intensity_min_edge','2d_intensity_max_edge','2d_intensity_mean_edge'
-                ]
-for y_col in feature_list:
-    fig,ax = plt.subplots(figsize=(4,3))
-
-    for colony in ['small','medium','large']:
-        
-        dfsub = dfm[dfm['colony']==colony].copy()
-        dfsub.dropna(subset=[y_col],inplace=True)
-
-        # create a pivot of the dataframe to get a 2d array of track_id x timepoint with each value being the density
-        pivot = dfsub.pivot(index=x_col, columns=column_val, values=y_col)
-        pivot.head()
-
-        mean = pivot.median(axis=1)
-        lower = pivot.quantile(0.05,axis=1)
-        upper = pivot.quantile(0.95,axis=1)
-        
-        xscale_factor, xlabel, xunit, xlimit = get_plot_labels_for_metric(x_col)
-        x = mean.index * xscale_factor
-        yscale_factor, ylabel, yunit, ylimit = get_plot_labels_for_metric(y_col)
-        y = mean.values * yscale_factor
-        yl = lower.values * yscale_factor
-        yu = upper.values * yscale_factor
-        
-        ax.plot(x, y, label=COLONY_LABELS[colony], color=COLONY_COLORS[colony])
-        ax.fill_between(x, yl, yu, alpha=0.2, color=COLONY_COLORS[colony],
-                        edgecolor='none')
-        ax.set_xlabel(f"{xlabel} {xunit}")
-        ax.set_ylabel(f"{ylabel} {yunit}\n(90% interpercentile range)")
+import matplotlib
+matplotlib.rcParams.update({'font.size': 8})
+matplotlib.rcParams.update({'axes.titlesize': 8})
+matplotlib.rcParams.update({'axes.labelsize': 8})
+matplotlib.rcParams.update({'xtick.labelsize': 8})
+matplotlib.rcParams.update({'ytick.labelsize': 8})
+matplotlib.rcParams.update({'legend.fontsize': 7})
+matplotlib.rcParams.update({'legend.title_fontsize': 8})
+matplotlib.rcParams.update({'figure.titlesize': 8})
+# make axis linewidth thinner
+matplotlib.rcParams.update({'axes.linewidth': 0.5})
 
 
-    ax.legend(loc="upper right", handletextpad=0.7, frameon=False)
-    plt.tight_layout()
-    for ext in ['.png','.pdf']:
-        save_and_show_plot(
-            f"{figdir}/{y_col}_vs_{x_col}_by_colony",
-            file_extension=ext,
-            dpi=300,
-            transparent=False,
-        )
-    plt.show()
-
 #%%
 # plot density as a function of nucleus size (and compare to old density metric)
```

Comment:
I think the most important thing here is that this new metric does not contain the source pitfalls that the old one did. Not that it doesnt correlate with the old density itself. 

- I would prefer that we just focus this workflow on the nucleus size and rename it as such.
- Showing that this results in a meaningful difference in the density is nice, but might be better proved by the correlation of old density with size. Not new density with old density. 
- Also since this is new "density" we need to keep the naming clear. 

Suggest calling this workflow: 
`check_for_size_dependence.py`

Reply:
good idea! renaming will make this comment dissapear. So commenting now before its gone :)